tests/test_main.py

```diff
@@ -0,0 +1,346 @@
+import json
+import time
+from unittest.mock import patch
+
+import pytest
+from fastapi.testclient import TestClient
+from sqlalchemy import create_engine
+from sqlalchemy.orm import exc
+from sqlalchemy.orm import sessionmaker
+
+#import main
+from auth.jwt_handler import sign_jwt
+from base_models.band_models import PostBandRequest
+from database_models.models import User, EmailVerification, Base, Band, BandMember, BandInvite
+from main import get_database, app
+from security.security import hash_password
+
+DATABASE_URL = "sqlite:///./test_database.db"
+engine = create_engine(
+    DATABASE_URL, connect_args={"check_same_thread": False}
+)
+
+Base.metadata.drop_all(bind=engine)
+Base.metadata.create_all(bind=engine)
+DbSession = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+client = TestClient(app)
+
+
+def override_get_db():
```

Comment:
We’ll figure out a better way to hook up a test db.

Reply:
This is the recommended way from https://fastapi.tiangolo.com/advanced/testing-database/